device/LIFX/controller.js

```diff
@@ -0,0 +1,85 @@
+/*
+ * Quick example for an LIFX light wrote by Geert Wille
+ */
+
+'use strict';
+
+const BluePromise = require('bluebird');
+
+let lifx = require('lifx-http-api');
+let client = new lifx({
+  bearerToken: 'INSERT_ACCESS_TOKEN_HERE' // https://cloud.lifx.com/settings
+});
+
+let sliderValue = 50;
+let switchValue = true;
+let sendComponentUpdate;
+
+/*
+ * Device Controller
+ * Events on that device from the Brain will be forwarded here for handling.
+ */
+module.exports.onPulse = function (deviceId, name) {
+  // Just randomly pulse the light
+  client.pulse('id:' + name, {
```

Comment:
is `'id:' + name` correct here?

can you make a helper function to get device selector, for example

```
function getLifxDeviceSelector(deviceId) {
  return 'id:' + deviceId;
}
```

this would make the code much more readable